blockdag/dag.go

```diff
@@ -27,6 +27,8 @@ const (
 	// maxOrphanBlocks is the maximum number of orphan blocks that can be
 	// queued.
 	maxOrphanBlocks = 100
+
+	isDAGCurrentMaxDiff = 12 * time.Hour
```

Comment:
TargetTimePerBlock and FinalityInterval are in dagparams. I argue that this should be in dagparams too.